pyfr/plugins/pseudostats.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+
+from pyfr.mpiutil import get_comm_rank_root
+from pyfr.plugins.base import BasePlugin, init_csv
+
+
+class PseudoStatsPlugin(BasePlugin):
+    name = 'pseudostats'
+    systems = ['*']
+
+    def __init__(self, intg, cfgsect, prefix):
+        super().__init__(intg, cfgsect, prefix)
+
+        if intg.formulation != 'dual':
```

Comment:
Add generic support for limiting plugins to formulation (as we do at the moment with systems).